formatcp.php

```diff
@@ -243,9 +235,7 @@ function handleActions($seriesName)
                 $success = $format->deleteSubFormat($cardset);
                 if (!$success) {
                     $hasError = true;
-                    $errormsg .= "Can't delete {$cardset} from child formats";
-
-                    return;
+                    $errormsg .= "Can't delete {$cardset} from child formats<br />";
```

Comment:
Why does this have a <br/>?

Reply:
Because it's inside a foreach, so I assume at some points, you are gonna delete more than one cardset?